src/Compilers/CSharp/Portable/Microsoft.CodeAnalysis.CSharp.csproj

```diff
@@ -17,7 +17,7 @@
 
     <!-- Enable output of generated files back into our Generated directory -->
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
-    <CompilerGeneratedFilesOutputPath>Generated</CompilerGeneratedFilesOutputPath>
+    <CompilerGeneratedFilesOutputPath>$(MSBuildThisFileDirectory)\Generated</CompilerGeneratedFilesOutputPath>
```

Comment:
Can we add a note to the issue to undo this change and put a link to the issue here?

Reply:
We don't necessarily need to undo it, absolute path should work always

Reply:
Yeah, I'm fine with leaving it as absolute as in the project file it means the same thing ultimately.